tests/test_builtin_casters.cpp

```diff
@@ -98,6 +98,10 @@ TEST_SUBMODULE(builtin_casters, m) {
     m.def("i64_str", [](std::int64_t v) { return std::to_string(v); });
     m.def("u64_str", [](std::uint64_t v) { return std::to_string(v); });
 
+    // test_int_convert
+    m.def("int_passthrough", [](int arg) { return arg; });
+    m.def("int_passthrough_noconvert", [](int arg) { return arg; }, py::arg().noconvert());
```

Comment:
This kills ICC.

Reply:
This is also why I'm continuing to work with the compiler team to fix this.
```suggestion
    m.def("int_passthrough_noconvert", [](int arg) { return arg; }, py::arg{}.noconvert());
```

Reply:
Aaaaaah, of course!

Reply:
So it's just the new push, and not the last changes. Thanks; I'll push a fix!